pkg/tsdb/cloudwatch/models/http_error.go

```diff
@@ -8,6 +8,7 @@ type HttpError struct {
 	StatusCode int
 }
 
+// Note: an HttpError is a user facing error. It should not contain any sensitive information.
```

Comment:
this is more aspirational than anything. Maybe it doesn't really belong in this pr